src/rpc/parse_commands.cc

```diff
@@ -150,7 +142,7 @@ parse_command(target_type target, const char* first, const char* last) {
   // Find the last character that is part of this command, skipping
   // the whitespace at the end. This ensures us that the caller
   // doesn't need to do this nor check for junk at the end.
-  first = std::find_if(first, last, std::not1(command_map_is_space()));
+  first = std::find_if(first, last, [&](char c) { return !command_map_is_space(c); });
   
   if (first != last) {
     if (*first != '\n' && *first != ';' && *first != '\0')
```

Comment:
This should be `!command_map_is_newline(*first)`.

Reply:
Changed to `!command_map_is_newline`.